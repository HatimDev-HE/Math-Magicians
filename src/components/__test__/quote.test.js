import React from 'react';
import Renderer from 'react-test-renderer';
import Quote from '../Quote';

it('claculating renders right', () => {
  const quote = Renderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
