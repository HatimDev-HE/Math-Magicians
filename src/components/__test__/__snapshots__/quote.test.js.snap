// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`claculating renders right 1`] = `
<section
  className="quote-section"
  data-testid="quoteSection"
>
  <blockquote
    cite="https://kidadl.com/quotes/top-most-inspirational-math-quotes-to-inspire-your-students-to-see-the-positives"
  >
    <p>
      “There should be no such thing as boring mathematics.”
    </p>
    <figcaption>
      — Edsger W. Dijkstra.
    </figcaption>
  </blockquote>
</section>
`;
